synapse/lib/grammar.py

```diff
@@ -15,6 +15,8 @@
 formre = regex.compile(formrestr)
 tagrestr = r'(\w+\.)*\w+'
 tagre = regex.compile(tagrestr)
+basepropstr = r'[a-z_][a-z0-9]*(?:(\:\:|\:|\.)[a-z_][a-z0-9]*)*'
```

Comment:
The `::` in the regex for embeds looks like it is still allowing invalid props to be added:
```
cli> storm $lib.model.ext.addFormProp('risk:vuln', '_acti::test', $lib.list('str', $lib.dict()), $lib.dict(doc="Foo"))
Executing query at 2021/10/20 14:23:35.586
complete. 0 nodes in 7 ms (0/sec).
cli> storm risk:vuln:_acti::test=asdf
Executing query at 2021/10/20 14:23:40.245
risk:vuln:_acti::test=asdf
               ^
Syntax Error: No terminal defined for ':' at line 1 col 16.  Expecting one of: #, *, *[, comparison operator, whitespace, whitespace or comment, {, |```

Reply:
from implicit property pivots - I've removed it.